torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -2292,6 +2292,10 @@ static const std::vector<OperatorGeneratorArgs> opGenArgs1{
           push(stack, c10::Device(pop(stack).toStringRef()));
         },
         aliasAnalysisFromSchema()),
+    OperatorGeneratorArgs(
+        TORCH_SELECTIVE_SCHEMA("aten::device(str type, int index) -> Device"),
```

Comment:
can you try calling this `"aten::device.with_index(str type, int index) -> Device"` and see if that fixes it?

Reply:
good tip, just pushed a new revision, thanks!

My expectation is that all builds would fail because of that, no just mobile.